src/container/acrManager.ts

```diff
@@ -86,37 +80,27 @@ export class AcrManager {
             await this.azureAccount.waitForFilters();
             const registryPromises: Array<Promise<AcrRegistryQuickPickItem[]>> = [];
             for (const azureSubscription of this.azureAccount.filters) {
-                const client = new ContainerRegistryManagementClient(
+                const client: Registries = new ContainerRegistryManagementClient(
                     azureSubscription.session.credentials,
                     azureSubscription.subscription.subscriptionId!,
-                );
+                ).registries;
 
                 registryPromises.push(
-                    this.listAcrRegistries(client.registries, client.registries.list())
+                    Utility.listAzureResources<Registry>(client.list(), client.listNext)
```

Comment:
Does azure resrouces (Registries, Workspaces) share the same interface? If so, the input of listAzureResources only need the interface?